Plugins/Skinning/src/SkinningComponent.hpp

```diff
@@ -13,11 +13,16 @@
 #include <Engine/Component/Component.hpp>
 #include <Engine/Managers/ComponentMessenger/ComponentMessenger.hpp>
 
+namespace SkinningPlugin
+{
+    class SkinningPluginC;
+}
 
 namespace SkinningPlugin
 {
     class SKIN_PLUGIN_API SkinningComponent : public Ra::Engine::Component
     {
+        friend class SkinningPlugin::SkinningPluginC;
```

Comment:
Maybe it would make sense for the system class to give access to this data then (in the form of const ptrs) ?